app/experimenter/nimbus-ui/schema.graphql

```diff
@@ -549,6 +551,13 @@ type NimbusOutcomeType {
   metrics: [NimbusOutcomeMetricType]
 }
 
+type NimbusProjectType {
+  id: Int
+  name: String
+  slug: String
+  nimbusexperimentSet: [NimbusExperimentType!]!
```

Comment:
This is probably a stupid question, but why do we need anything more than `id` and `name` here? (or `name` and `slug`, I'm not sure the difference between slug and id)

Reply:
so the thing is Projects is the existing model, but here are some insights `Slug is a URL friendly short label for specific content. It only contain Letters, Numbers, Underscores or Hyphens. Slugs are commonly save with the respective content and it pass as a URL string.` whereas id is django generated unique keys, so we have name user friendly and slug as identifier